WordPress/src/main/res/layout/debug_settings_row.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    style="@style/MainBottomSheetRowLayout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/title"
+        style="@style/MainBottomSheetRowTextView"
```

Comment:
Can we change this to a bigger heading font?  It is indistinguishable from the other rows

Reply:
I didn't put too much thought into this UI, since it's something we're only going to use internally, but I'm not sure we actually need these rows to be different from the existing ones. We don't necessarily want these items to be highlighted, so I think it's probably fine to just reuse the current styling we already use across the app. Feel free to disagree though!

Reply:
The list is already long, It may only get longer as time passes by and more categories too.  So, if I'm looking for a setting while scrolling a more prominent heading might help find it quicker I guess while scrolling.  What do you think?

Reply:
I think this is a valid concern. I'm not convinced an increased font would solve this problem though, as other future items would still have the same font size. Maybe we can address this by including a different icon for each item? I'm open to other suggestions. Unfortunately, this is not something I can prioritize right now, so I'm just gonna go ahead and merge this, since you have already approved. Thanks again! 🙇‍♂️